core/trino-spi/pom.xml

```diff
@@ -35,6 +35,11 @@
             <optional>true</optional>
         </dependency>
 
+        <dependency>
```

Comment:
Adding dependency to SPI is serious. It needs to go in separate PR.

Reply:
It is, kind of. It can't be a separate commit entirely because then all the enforcers etc. will complain that the dependency is not used.

Reply:
Oh, sorry, you said "PR", I read "commit" :)

Reply:
Extracted as #15052

Reply:
As I understand, the extracted PR blocks this PR, right?

Reply:
In a sense, yes. You could still merge this one as it includes the other one and the result will be the same :)